src/inspector/models/score/scoredisplaysettingsmodel.cpp

```diff
@@ -106,8 +105,82 @@ void ScoreSettingsModel::setShouldShowPageMargins(bool shouldShowPageMargins)
         return;
     }
 
-    adapter()->updatePageMarginsVisibility(shouldShowPageMargins);
+    adapter()->updatePageMarginsVisibility();
+
+    updateShouldShowPageMargins(shouldShowPageMargins);
+}
+
+void ScoreSettingsModel::updateShouldShowInvisible(bool isVisible)
+{
+    m_shouldShowInvisible = isVisible;
```

Comment:
Better to add a check here (same for methods below) that the value is actually changed:

if (isVisible == m_shouldShowInvisible) {
return;
}

m_shouldShowInvisible = isVisible;
emit shouldShowInvisibleChanged(isVisible);